src/juju/model-selectors.ts

```diff
@@ -32,3 +48,51 @@ export function getModelUUIDByName(modelName: string, ownerName: string) {
     return modelUUID;
   });
 }
+
+export function getModelApplications(modelUUID: string) {
+  return createSelector(
+    getModelWatcherDataByUUID(modelUUID),
+    (modelWatcherData): ApplicationData | null => {
+      if (modelWatcherData) {
+        return modelWatcherData.applications;
+      }
+      return null;
```

Comment:
Again with all these you could save some precious lines with `return modelWatcherData?.applications ?? null`.

Reply:
Got to save those precious bytes!